flink-ml-lib/src/test/java/org/apache/flink/ml/classification/knn/KnnTest.java

```diff
@@ -0,0 +1,251 @@
+                            Row.of(Vectors.dense(2.5, 3.2), 5.0),
+                            Row.of(Vectors.dense(2.5, 3.2), 5.0),
+                            Row.of(Vectors.dense(2.1, 3.1), 1.0)));
+
+    private static final List<Row> testArray =
+            new ArrayList<>(
+                    Arrays.asList(
+                            Row.of(Vectors.dense(4.0, 4.1), 5.0),
+                            Row.of(Vectors.dense(300, 42), 2.0)));
+    private Table testData;
+
+    @Before
+    public void before() {
+        Configuration config = new Configuration();
+        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
+        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
+        env.setParallelism(4);
+        env.enableCheckpointing(100);
+        env.setRestartStrategy(RestartStrategies.noRestart());
+        tEnv = StreamTableEnvironment.create(env);
+
+        Schema schema =
+                Schema.newBuilder()
+                        .column("f0", DataTypes.of(DenseVector.class))
+                        .column("f1", DataTypes.DOUBLE())
+                        .build();
+
+        DataStream<Row> dataStream = env.fromCollection(trainArray);
+        trainData = tEnv.fromDataStream(dataStream, schema).as("features", "label");
+
+        DataStream<Row> predDataStream = env.fromCollection(testArray);
+        testData = tEnv.fromDataStream(predDataStream, schema).as("features", "label");
+    }
+
+    private static List<Tuple2<Double, Double>> executeAndCollect(
+            Table output, String labelCol, String predictionCol) throws Exception {
+        StreamTableEnvironment tEnv =
+                (StreamTableEnvironment) ((TableImpl) output).getTableEnvironment();
+
+        DataStream<Tuple2<Double, Double>> stream =
+                tEnv.toDataStream(output)
+                        .map(
+                                new MapFunction<Row, Tuple2<Double, Double>>() {
+                                    @Override
+                                    public Tuple2<Double, Double> map(Row row) {
+                                        return Tuple2.of(
+                                                (Double) row.getField(labelCol),
+                                                (Double) row.getField(predictionCol));
+                                    }
+                                });
+        return IteratorUtils.toList(stream.executeAndCollect());
+    }
+
+    private static void verifyClusteringResult(List<Tuple2<Double, Double>> result) {
+        for (Tuple2<Double, Double> t2 : result) {
+            Assert.assertEquals(t2.f0, t2.f1);
+        }
+    }
+
+    @Test
+    public void testParam() {
+        Knn knn = new Knn();
+        assertEquals("features", knn.getFeaturesCol());
+        assertEquals("label", knn.getLabelCol());
+        assertEquals(10L, knn.getK().longValue());
```

Comment:
Can you remove the `.longValue()` here and simply use `assertEquals(10, knn.getK());`?

Reply:
Function assertEquals(Integer a, Integer b) is not exist.